requirements.in

```diff
@@ -1,7 +1,7 @@
-stdlib-list
-cached-property
-toml
+# Ourselves, including our dependencies
+-e file:.#z3c.dependencychecker[test]
```

Comment:
I did not know that you could do it this way, I was always using the shorthand `-e .[test]` 😄 

Is there any benefit on stating this `file:.#z3c.dependencychecker` ? 🤔

Reply:
I think it was for some other tool... That way, the installation mechanism knows beforehand which package it is going to be dev-installing. I might remember the reason in a few weeks.